recipes/wait_for2/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "wait_for2" %}
+{% set version = "0.3.2" %}
+{% set python_min = "3.7" %}
```

Comment:
Only set `python_min` if the minimum required version is higher than the conda-forge default. 

```suggestion
```

Reply:
You have to had only to remove this line. Please revert all the other changes.